### PR TITLE
feat: use zero trust dialer again

### PR DESCRIPTION
**Pull request by dovholuknf**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [ ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

File: bootstrap/registration/registry.go [hunk 1]
```diff
@@ -69,10 +68,10 @@ func createRegistryClient(
 		if err != nil {
 			return nil, err
 		}
-
+		// TODO: address this issue in some way? the openziti fallback dialer should be dialing this address if not zitified
 		// Bypass the zero trust zitidfied transport for Core Keeper Registry client
 		// Should leverage the HttpTransportFromService function from zerotrust pkg, set the default transport for now
-		secretProvider.SetHttpTransport(http.DefaultTransport)
+		// secretProvider.SetHttpTransport(http.DefaultTransport)
```
Comment (cloudxxx8) [inline bootstrap/registration/registry.go:74]:
```
if (bootstrapConfig.Registry.Type == "keeper") {
  secretProvider.SetHttpTransport(http.DefaultTransport)
}
```

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]

**Review by cloudxxx8 — APPROVED** [review]
LGTM

File: bootstrap/registration/registry.go (hunk 1 above)
Comment (dovholuknf) [reply bootstrap/registration/registry.go]: updated

**Comment by dovholuknf** [comment]
@cloudxxx8 -- i removed a second location where the bypass was put in place. i didn't see and equivalent of:

> bootstrapConfig.Registry.Type == "keeper"

so i removed it. Maybe I missed it?

**Comment by cloudxxx8** [comment]
close this PR due to the byPass code is removed from https://github.com/edgexfoundry/go-mod-bootstrap/pull/736

**Pull request closed without merging**